rust/src/rest.rs

```diff
@@ -29,19 +29,26 @@ pub async fn serve(addr: impl Into<SocketAddr> + 'static, handle: Handle) {
         .and(warp::get())
         .and(part_pk())
         .and(with_hdl(handle.clone()))
-        .and_then(handle_seeds)
-        .recover(handle_reject);
+        .and_then(handle_seeds);
+        // .recover(handle_reject);
```

Comment:
feel free to remove this comment

Reply:
thanks for catching this! seems that i forgot about it.